Resources/Prototypes/_LostParadise/Entities/Markers/Spawners/jobs.yml

```
@@ -1,14 +1,14 @@
-#     job_id: LPPAdjutant
-#   - type: Sprite
-#     layers:
-#     - state: green
-#     - state: hop
+- type: entity
+  id: LPPAdjutantSpawn
+  parent: SpawnPointJobBase
+  name: adjutant spawn
+  components:
+  - type: SpawnPoint
+    job_id: AdministrativeAssistant # LPPAdjutant # Появился AdministrativeAssistant
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)